notebooks/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>What would be a recommended number of negative instances (between X and Y)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
For training, the recommended number is 4. For evaluation, it depends on your dataset. I will add descriptions to the notebook.